### ISSUE TITLE
New "Defense mode"

### ISSUE THREAD

**Issue by Seple** [ermiyaeskandary/Slither.io-bot#78]
Defense Mode - Currently, this option is good if you want someone to lure the circle surround us and fast surprise attack and quickly jump on the "Quick Boost" before him.

I think that the better:
Cut back on the body.

![image](https://cloud.githubusercontent.com/assets/18369104/15807917/3f1282da-2b61-11e6-8e10-de30e8618d68.png)

There is no possibility headshot.

**Comment by dracco1993** [comment]
So you're saying just go back and forth along your own body?

**Comment by Seple** [comment]
Yes. The Best head protection. I think that the better. The simplest solutions are the best ;)

I'm sorry for my language english is not good.

**Comment by tjbcg** [comment]
Except when you reach the end, your head will be forced to leave your body. I think the best defence is the circle one, but we need to make the circle bigger. (this way the head stays safe)

**Comment by Seple** [comment]
(just a suggestion) 
...do not turn in a perfect circle always head protrudes.
While the circle is small and vulnerable (exposes himself) to attack
They were the best mix "back on the body" and "Circle" ....or fast intelligent manual control buton mouse #54

**Comment by ermiyaeskandary** [comment]
Ah - forgot #54 
Sorry... will start working on it soon

**Comment by Seple** [comment]
@ErmiyaEskandary  calmly not burn

**Comment by ermiyaeskandary** [comment]
@Seple 

> calmly not burn

??

**Comment by Seple** [comment]
"calmly, not burn (fire)" - proverb - I do not know how to write in English Sorry

**Comment by ChadSki** [comment]
Reminds me of "haste makes waste". :) What does the untranslated proverb look like?

**Comment by Seple** [comment]
@ChadSki (Jesteś Polak? - understand me?) You come from Poland?

**Comment by ChadSki** [comment]
I have Polish ancestry but do not speak it. The only word I know is "kapcie" (slippers) from my Grandpa. But languages are interesting! And it's easier to look up meanings from the original text.

**Comment by Seple** [comment]
I understand. I wanted help my to translate. I'm sorry for my language english is not good. 
Regards grandfather ;)

**Comment by ChadSki** [comment]
No worries! Your English is far better than my Polish! :laughing:

**Comment by Seple** [comment]
You can make a perfect circle without compromising the head? 
Yes or not?

**Comment by ermiyaeskandary** [comment]
I don't think so as it tightens. We can do what you said though - go over your own body. Theoretically it's possible.

**Comment by ermiyaeskandary** [comment]
We can make a bigger circle like what @FliiFe said.

**Comment by chancity** [comment]
Go through your own snakes dying pts,  I'll write code it up sometime today

**Comment by Seple** [comment]
Super! :+1:

**Comment by ermiyaeskandary** [comment]
@chancity Please please make a PR for this...

**Comment by Seple** [comment]
Very nice please work
